src/containers/org/settings/OrgSettingsContainer.js

```diff
@@ -197,28 +201,44 @@ const OrgSettingsContainer = ({
                 onClick={() => clipboardWriteText(organizationId)} />
           </ActionsGrid>
         </div>
-        <div>
-          <Typography component="h2" variant="body2">DATABASE NAME</Typography>
-          <ActionsGrid align={{ v: 'center' }} fit>
-            <Pre>{databaseName}</Pre>
-            <CopyButton
-                aria-label="copy database name"
-                onClick={() => clipboardWriteText(databaseName)} />
-            <EditButton
-                aria-label="edit database name"
-                color="default"
-                onClick={() => setIsVisibleRenameModal(true)} />
-          </ActionsGrid>
-        </div>
-        <div>
-          <Typography component="h2" variant="body2">JDBC URL</Typography>
-          <ActionsGrid align={{ v: 'center' }} fit>
-            <Pre>{jdbcURL}</Pre>
-            <CopyButton
-                aria-label="copy jdbc url"
-                onClick={() => clipboardWriteText(jdbcURL)} />
-          </ActionsGrid>
-        </div>
+        {
+          isUnauthorized
+            ? (
+              <Typography color="error" variant="overline">
+                You are not authorized to view database credetials.
+              </Typography>
+            )
+            : (
+              <>
+                <div>
+                  <Typography component="h2" variant="body2">DATABASE NAME</Typography>
```

Comment:
The request to get the database name only checks if the user has READ on the org, so I believe any member will be able to view the database name. However, to get the integration account, you must have OWNER on the org, so you'll have to split these up.

Reply:
Well, the database name comes through in the IntegrationDetails, which is no populated if the `getIntegrationDetails` call fails. That's why I hid it.

Reply:
The database name is a separate request being made in the same saga. The saga currently fails if either request fails, which needs to change.

Reply:
ok, moved the return of integration details to the finally block